pkg/eestream/decode.go

```diff
@@ -102,16 +102,29 @@ func (dr *decodedReader) Close() error {
 	errorThreshold := len(dr.readers) - dr.scheme.RequiredCount()
 	dr.close.Do(func() {
 		var errlist errs.Group
```

Comment:
Let's make a nice type for it in `errs2.` e.g. `errs2.WaitGroup` or something.

Reply:
added `errs2.Group`